code/game/gamemodes/objective.dm

```diff
@@ -246,6 +265,10 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	martyr_compatible = FALSE
 
 /datum/objective/maroon/update_explanation_text()
+	if(delayed_objective)
+		explanation_text = "Your objective is to make sure another crewmember doesn't leave on the Escape Shuttle. You will receive further information in a few minutes."
+		return
+
```

Comment:
"The one thing I could change is make it into variables on the objective and have every update_explanation_text call parent"

Reply:
It is done